content/overview/quick-start.md

```diff
@@ -55,22 +55,22 @@ pip install 'farm-haystack[all]' ## or 'all-gpu' for the GPU-enabled dependencie
 {{< /tab >}}
 {{< /tabs >}}
 
-For a more comprehensive installation guide, inlcuding methods for various operating systems, refer to our documentation.
+For a more comprehensive installation guide, including methods for various operating systems, refer to our documentation.
 
 {{< button url="https://docs.haystack.deepset.ai/docs/installation" text="Docs: Installation" color="green">}}
 
-## Build Your First RAG Pipeline
+## Build Your First Retrieval Augmented Generation (RAG) Pipeline
 
-Haystack is built around the concept of pipelines. A pipeline is a powerful structure made up of components that can be used to perform a task.
-For example, you can connect together a Retriever and a PromptNode to build a Generative Question Answering pipeline.
+Haystack is built around the concept of pipelines. A pipeline is a powerful structure that performs an NLP task. It's made up of components chained together.
+For example, you can connect a Retriever and a PromptNode to build a Generative Question Answering pipeline that uses your own data.
 
-Try out how Haystack answers questions about Game of Thrones using **Retrieval Augmented Generation (RAG)** approach 👇
+Try out how Haystack answers questions about Game of Thrones using the **RAG** approach 👇
 
 Install Haystack in the minimal form:
```

Comment:
Does "minimal form" mean the most basic installation?

ah ok, I get it now, you can say: Run the minimal Haystack installation: